app/routes.py

```diff
@@ -0,0 +1,302 @@
+
+        return make_response({"id": new_rental.id,
+                                "customer_id": new_rental.customer_id,
+                                "video_id": new_rental.video_id,
+                                "available_inventory": available_inventory_after_check_out,
+                                "videos_checked_out_count": videos_checked_out_count})
+
+@rental_bp.route('/check-in', methods=['POST'])
+def handle_checkin():
+    request_body = request.get_json()
+
+    if "customer_id" not in request_body.keys():
+        return make_response ({"details": "Customer ID required."}, 400)
+
+    elif Customer.query.filter_by(id=request_body['customer_id']).first() is None:
+        return make_response ({"details": "Customer not found."}, 404)
+
+    elif 'video_id' not in request_body.keys():
+        return make_response({"details": "Request must include video id."}, 400)
+
+    elif Rental.query.filter_by(video_id=request_body['video_id']).first() is None and \
+            Rental.query.filter_by(customer_id=request_body['customer_id']).first() is None:
+        return make_response({
+            "message": f"No outstanding rentals for customer {request_body['customer_id']} and video {request_body['video_id']}"}, 400)
+    
+    elif Video.query.filter_by(id=request_body['video_id']).first() is None:
+        return make_response({"details": "Video not found"}, 404)
+
+    else:
+        # Using the information from request_body, locate the rental whose status needs to be changed (i.e., the video is being returned)
+        existing_rental = Rental.query.filter_by(video_id=request_body["video_id"], customer_id=request_body["customer_id"], \
+                            is_checked_in=False).first()
+
+        existing_rental.is_checked_in = True
+
+        db.session.commit()
+
+        available_inventory_after_checkin = Video.query.filter_by(id=request_body['video_id']).first().calculate_available_inventory()
+        videos_checked_out_count = Customer.query.filter_by(id=request_body['customer_id']).first().get_videos_checked_out_count()
+
+        return make_response({"video_id": existing_rental.video_id,
+                                "customer_id": existing_rental.customer_id,
+                                "videos_checked_out_count": videos_checked_out_count,
+                                "available_inventory": available_inventory_after_checkin})
+
+
+
+
+
+
+    
+    
+    # new_rental = Rental(video_id=request_body["video_id"],
+    #                     customer_id=request_body["customer_id"])
+    # updated_videos_checked_out = Customer.query.filter_by(id=request_body['customer_id']).first().videos_checked_out_count +1   
+    # Customer.query.filter_by(id=request_body['customer_id']).first().videos_checked_out_count 
+
+
+
+    # return make_response({"customer_id": request_body["customer_id"] ,
+    #                           "video_id": request_body["video_id"]})
+
```

Comment:
remove unused code when pushing final code. If you want to keep this code, I would move it to a readME file.